.pipeline/blubber.yaml

```diff
@@ -0,0 +1,23 @@
+version: v3
+base: docker-registry.wikimedia.org/nodejs-slim
+lives:
+  in: /srv/service
+runs:
+  environment: { APP_BASE_PATH: /srv/service }
+
+variants:
+  build:
+    base: docker-registry.wikimedia.org/nodejs-devel
+    apt: { packages: [git, build-essential, python-pkgconfig] }
+    node: {requirements: [package.json]}
```

Comment:
Also add `runs: { environment: { LINK: g++ } }`. Oh, and a nit: please put a space after/before the braces.